.github/CHANGELOG.md

```diff
@@ -66,6 +66,9 @@
 
 ### Documentation
 
+*  Update ``lightning.tensor`` documentation on all the new features.
```

Comment:
Perhaps specify which features those are, I could imagine looking at this entry in one year and not knowing what was meant

Reply:
Done.